src/coreclr/jit/fgopt.cpp

```diff
@@ -3412,9 +3412,24 @@ bool Compiler::fgReorderBlocks(bool useProfile)
         }
     }
 
-    // If we will be reordering blocks, ensure the false target of a BBJ_COND block is its next block
     if (useProfile)
     {
+        if (JitConfig.JitDoReversePostOrderLayout())
+        {
+            fgDoReversePostOrderLayout();
+            fgMoveColdBlocks();
+
+#ifdef DEBUG
```

Comment:
You can just remove this whole `#ifdef DEBUG` bit, we will check the BB list at end of phase, which is imminent.